moninag/static/src/ts/app/servers/servers.component.html

```diff
@@ -1,26 +1,17 @@
-<div class="table-responsive">
-    <ul style="list-style-type:none; padding:0">
-        <table class="table">
-            <thead>
-                <tr>
-                    <th></th>
-                    <th>Name</th>
-                    <th>Address</th>
-                    <th>State</th>
-                    <th></th>
-                </tr>
-            </thead>
-            <tbody>
-                <tr *ngFor="let server of servers">
-                    <td>{{ server.name }}</td>
-                    <td>{{ server.address }}</td>
-                    <td>{{ server.state }}</td>
-                    <td>
-                        <button (click)="onSelect(server)" (click)="gotoEdit()" value={{server.id}}>Edit</button>
-                    </td>
-                </tr>
-            </tbody>
-        </table>
-    </ul>
-    <button (click)="onSelect(server)" (click)="add()">Add</button>
-</div>
+<md-card>
+            <md-card-title>
+                Your Servers
+           </md-card-title>
+            <md-card-content>
+                <button md-button (click)="onSelect(server)" (click)="add()">Add</button>
+              <md-card fxLayoutWrap fxLayout="row" fxLayoutAlign="space-between center">
+                <div *ngFor="let server of servers">
+                  <md-card (click)="onSelect(server)" (click)="gotoEdit()">
+                    Name: {{ server.name }}
+                    Adress: {{ server.address }}
+                    State: {{ server.state }}
+                  </md-card>
```

Comment:
You display information about server in one row.
Maybe it is better to display in column? :nerd_face: